app/views/pages/home.html.erb

```diff
@@ -1,72 +1,106 @@
-<div class="wrapper-features">
-  <h2 class="text-center">How it works</h2>
-  <div class="features">
+  <div class="use-case hidden-xs">
+    <h3>Some places choosen with Choogle</h3>
     <div class="container">
-      <div class="row">
-        <div class="col-xs-12 col-sm-4">
-          <div class="feature">
-              <h6>☝️</h6>
-              <h3>Suggest options</h3>
-              <p>Select a first place to go, a date and a timer. Participants will have the option to add other suggestions.</p>
+      <div id="slider" class="carousel slide col-xs-8  padd-lt0" data-ride="carousel" data-interval="10000" data-pause="hover">
+        <div class="carousel-inner">
+          <div class="item active">
+              <img alt="First slide"4 src="assets/festival.jpg" >   
           </div>
-        </div>
-        <div class="col-xs-12 col-sm-4">
-          <div class="feature">
-            <h6>👫</h6>
-            <h3>Invite participants to vote</h3>
-            <p>They can now upvote their favorite places. They don’t even need an account.</p>
+          <div class="item">
+              <img alt="Second slide" src="assets/afterwork.jpg">  
+          </div>
+          <div class="item">
+              <img alt="Third slide" src="assets/weekend.jpg">
           </div>
         </div>
-        <div class="col-xs-12 col-sm-4">
-          <div class="feature">
-            <h6>🎯</h6>
-            <h3>Select the best option</h3>
-            <p>Once the timer is over pick the final option and received all its informations.</p>
+      </div>
+
+      <div id="slider_captions" class="col-xs-4">
+        <div>
+          <div id="caption-0" class="carousel-caption">
+              <h2>Liverpool International Music Festival 2017, Liverpool</h2>
+              <div class="tag">
+                <h5 class="tags tags-color8">free</h5><h5 class="tags tags-color6">outdoors</h5><h5 class="tags tags-color7">sefton park</h5><h5 class="tags tags-color4">gorgon city</h5><h5 class="tags tags-color2">corinne bailey rae</h5><h5 class="tags tags-color1">so solid crew</h5><h5 class="tags tags-color3">rock</h5>
+              </div>
+              <p>Upvoted by Amelia, Oliver, Jack, George, Ava, Henry, Emily and Jacob.</p>
+              <p>Proposed by Ava for the <strong>Annual festival trip.</strong></p>
+
+          </div>
+          <div id="caption-1" class="carousel-caption">
+              <h2>Rosa Bonheur Bar, Belleville, Paris</h2>
+              <div class="tag">
+                <h5 class="tags tags-color3">guinguette</h5><h5 class="tags tags-color1">metro 7</h5><h5 class="tags tags-color4">metro 11</h5><h5 class="tags tags-color7">belleville</h5><h5 class="tags tags-color2">outdoor</h5><h5 class="tags tags-color6">rosé</h5><h5 class="tags tags-color8">tapas</h5><h5 class="tags tags-color7">pizza</h5>
+              </div>
+              <p>Upvoted by Lisa, Romain, Eliott, Simon, Marie, Vincent, Tatiana and Sylvain.</p>
```

Comment:
;)

Reply:
That's a great "du dur" section :)

Reply:
I was exactly commenting this. Du dur is great for this section of the site. Too many variables leans to a Russian roulette home page